docs/manifest.json

```diff
@@ -517,7 +517,7 @@
 	},
 	{
 		"title": "Understanding global state, local context and derived state",
-		"slug": "undestanding-global-state-local-context-and-derived-state",
+		"slug": "understanding-global-state-local-context-and-derived-state",
```

Comment:
I'm concerned that changing this slug may have ramifications for syncing with the Block Editor Handbook.

cc @ndiego @juanmaguitar